support-frontend/assets/helpers/productPrice/priceDescriptions.js

```diff
@@ -141,18 +134,17 @@ function getPriceDescription(
       promotion.discountedPrice,
       promotion.numberOfDiscountedPeriods,
       billingPeriod,
-      compact,
     );
   }
-  return standardRate(glyph, productPrice.price, billingPeriod, productPrice.fixedTerm, compact);
+  return standardRate(glyph, productPrice.price, billingPeriod, productPrice.fixedTerm);
 }
 
 function getAppliedPromoDescription(billingPeriod: BillingPeriod, productPrice: ProductPrice) {
   const appliedPromo = getAppliedPromo(productPrice.promotions);
   if (appliedPromo && appliedPromo.landingPage && appliedPromo.landingPage.roundel) {
     return appliedPromo.landingPage.roundel;
   }
-  return '';
+  return billingPeriod === Annual ? 'Save over 20% against monthly in the first year.' : '';
```

Comment:
I'm not sure that this default should be here, this function can be used by any product but this description only applies to DS

Reply:
Yes, fair call